### DISCUSSION TITLE
Frage zu Aufgabe 3

### DISCUSSION THREAD

**Discussion by simsek-beyza** [irgroup-classrooms/dis08-2021#68] [Q&A]
Hallo zusammen,

wir wollten einmal nachfragen, was man machen muss, wenn nicht jede URL in der CSV-Datei eine Infobox besitzt? Wäre das ein Problem?  Vielen Dank im Voraus

Viele Grüße, 
Data Deputy

**Comment by phschaer** [comment]
Nicht unbedingt, solange Sie damit entsprechend in Ihrem Code umgehen, ist alles gut. Wenn es jedoch kaum Infoboxen zu ihrem Thema gibt, ggf. nochmal was anderes ausprobieren. Aber wie gesagt: Im Prinzip kein Problem.
